assert/assertions.go

```diff
@@ -183,22 +184,39 @@ func indentMessageLines(message string, tabs int) string {
 
 // Fail reports a failure through
 func Fail(t TestingT, failureMessage string, msgAndArgs ...interface{}) bool {
-
+	location := ""
+	printLocation, _ := os.LookupEnv("TESTIFY_PRINT_LOCATION")
```

Comment:
Thanks, I will change it once I get back to my dev environment (I am currently traveling).